src/C++/FieldMap.h

```diff
@@ -184,6 +184,15 @@ class FieldMap
     return &getFieldRef( tag );
   }
 
+  /// Get direct access to a field through a pointer if set
+  const FieldBase* const getFieldPtrIfSet( int tag )
```

Comment:
These methods seem pretty dangerous to me. Seems like it would be easy to propagate null pointers into an application.

Reply:
the intention is to made a exception-free path without two identical map lookups (isSetField / getField) and make it more efficient. Also, the code is not use anywhere in quickfix standard base internally, so no internal A/V (NPE) was created internally.

Reply:
Also, resolved conflicts with poll() changes in master branch